draft-ietf-ppm-dap.md

```diff
@@ -4215,665 +4226,23 @@ component is using. This MAY be used as a hint by the receiver of the request
-
-: N/A
-
-Additional information:
-
-: <dl>
-  <dt>Magic number(s):</dt><dd>N/A</dd>
-  <dt>Deprecated alias names for this type:</dt><dd>N/A</dd>
-  <dt>File extension(s):</dt><dd>N/A</dd>
-  <dt>Macintosh file type code(s):</dt><dd>N/A</dd>
-  </dl>
-
-Person and email address to contact for further information:
-
-: see Authors' Addresses section of the published specification
-
-Intended usage:
-
-: COMMON
-
-Restrictions on usage:
-
-: N/A
-
-Author:
-
-: see Authors' Addresses section of the published specification
-
-Change controller:
-
-: IESG
-
-### "application/dap-upload-req" media type
-
-Type name:
-
-: application
-
-Subtype name:
-
-: dap-upload-req
-
-Required parameters:
-
-: N/A
-
-Optional parameters:
-
-: None
-
-Encoding considerations:
-
-: only "8bit" or "binary" is permitted
-
-Security considerations:
-
-: see {{upload-flow}} of the published specification
-
-Interoperability considerations:
-
-: N/A
-
-Published specification:
-
-: RFC XXXX
-
-Applications that use this media type:
-
-: N/A
-
-Fragment identifier considerations:
-
-: N/A
-
-Additional information:
-
-: <dl>
-  <dt>Magic number(s):</dt><dd>N/A</dd>
-  <dt>Deprecated alias names for this type:</dt><dd>N/A</dd>
-  <dt>File extension(s):</dt><dd>N/A</dd>
-  <dt>Macintosh file type code(s):</dt><dd>N/A</dd>
-  </dl>
-
-Person and email address to contact for further information:
-
-: see Authors' Addresses section of the published specification
-
-Intended usage:
-
-: COMMON
-
-Restrictions on usage:
-
-: N/A
-
-Author:
-
-: see Authors' Addresses section of the published specification
-
-Change controller:
-
-: IESG
-
-### "application/dap-upload-resp" media type
-
-Type name:
-
-: application
-
-Subtype name:
-
-: dap-upload-resp
-
-Required parameters:
-
-: N/A
-
-Optional parameters:
-
-: None
-
-Encoding considerations:
-
-: only "8bit" or "binary" is permitted
-
-Security considerations:
-
-: see {{upload-flow}} of the published specification
-
-Interoperability considerations:
-
-: N/A
-
-Published specification:
-
-: RFC XXXX
-
-Applications that use this media type:
-
-: N/A
-
-Fragment identifier considerations:
-
-: N/A
-
-Additional information:
-
-: <dl>
-  <dt>Magic number(s):</dt><dd>N/A</dd>
-  <dt>Deprecated alias names for this type:</dt><dd>N/A</dd>
-  <dt>File extension(s):</dt><dd>N/A</dd>
-  <dt>Macintosh file type code(s):</dt><dd>N/A</dd>
-  </dl>
-
-Person and email address to contact for further information:
-
-: see Authors' Addresses section of the published specification
-
-Intended usage:
-
-: COMMON
-
-Restrictions on usage:
-
-: N/A
-
-Author:
-
-: see Authors' Addresses section of the published specification
-
-Change controller:
-
-: IESG
-
-### "application/dap-aggregation-job-init-req" media type
+be (RFC EDITOR: replace "will be" with "has been") updated to include the
+`application/dap` media type.
 
 Type name:
 
 : application
 
 Subtype name:
 
-: dap-aggregation-job-init-req
+: dap
```

Comment:
"dap" seems likely to have name collisions. Perhaps "ppm-dap" would be a better name.